modules/edge-identity/src/lib.rs

```diff
@@ -299,7 +299,7 @@ mod tests {
 			assert_ok!(attest_to_identity(public, identity_hash, attestation));
 
 			let verifier = H256::from(9);
```

Comment:
No new test cases to try out the feature?

Reply:
So far, nope haha. I will add some for the new changes. Wanted to get more feedback first.